mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -5012,20 +5030,315 @@ msgstr "She means well..."
+
+msgid "doorchosen"
+msgstr "But I guess we better focus on the Battle Area, since we're here and all."
+
+msgid "doorchosen2"
+msgstr "I'll start with door 3! Lucky number 3 after all!"
+
+msgid "doorchosen3"
+msgstr "That girl will be the death of us."
+
+msgid "doorchosen4"
+msgstr "I'll start with door 2. Good Luck."
+
+msgid "notyettababa"
+msgstr "Only one person can challenge an Acolyte at a time. Please challenge another one."
+
+msgid "notyettababafourth"
+msgstr "This is the most powerful Acolyte before the Master, please finish challenging the others first."
+
+msgid "itslocked"
+msgstr "The doors are locked, maybe you should try defeating all 4 Acolytes first?"
+
+msgid "healspls"
+msgstr "Give me your Tuxemon."
+
+msgid "healsdone"
+msgstr "Your Tuxemon have been healed."
+
+msgid "areyoureadytorumble"
+msgstr "Are you ready to battle the Acolyte?"
+
+msgid "areyoureadytorumbleno"
+msgstr "Very well, come back once you're prepared."
+
+msgid "areyoureadytorumbleyes"
+msgstr "Very well, before we fetch the Acolyte, we'll heal your Tuxemon."
+
+msgid "areyoureadytorumbleyes2"
+msgstr "There, now that you actually stand a chance, we will fetch the Acolyte."
+
+msgid "uhhjason"
+msgstr "Uhh... Jason?"
+
+msgid "uhhjason2"
+msgstr "Shoot! He always does this..."
+
+msgid "uhhjason3"
+msgstr "JASON!"
+
+msgid "challenger"
+msgstr "Challenger!\n"
+"Step to the center and face me!"
+
+msgid "challenger2"
+msgstr "You have done well to make it this far!"
+
+msgid "challenger3"
+msgstr "... .... ..."
+
+msgid "challenger4"
+msgstr "Wait, this is your first battle?\n"
+"I was told I would be the third battle..."
+
+msgid "challenger5"
+msgstr "No matter!"
+
+msgid "challenger6"
+msgstr "This just means it will be easier to crush you!"
+
+msgid "challenger7"
+msgstr "Ready yourself Challenger! It's time... TO FIGHT!"
+
+msgid "tabacongratsfirst"
+msgstr "... ... Congrats on the win..."
+
+msgid "tabacongratsfirst2"
+msgstr "I definitely wasn't expecting a performance like that. Kyle told us all that you would be an easy win.\n"
+"I guess he meant for you and not us!"
+
+msgid "tabacongratsfirst3"
+msgstr "The other three acolytes are more powerful than me, so you may want to try and fight the wild Tuxemon in the passageways a bit.\n"
+"Other than that, Good Luck!\n"
+"If you walk through the entrance behind me, you'll be teleported back to the main room."
+
+msgid "acolyte1loss"
+msgstr "Ha! Some Challenger you are. You'll have to do better than that to beat me!\n"
+"Try and level up your Tuxemon before facing me again!"
+
+msgid "acolyte1redo"
+msgstr "So, you actually crawled back... Let's see how powerful you really are!"
+
 msgid "ripjimmy"
 msgstr "The grave says:\n"
 "Here lies Jimmy. A friend, a knight, and ... bad person overall. However, ... never ... ... Help! ...\n"
 "The rest is weathered away."
 
-msgid "tabanurse_dialog\n"
-msgstr "Welcome to the ${{map_name}} Tuxecenter!\n"
+msgid "tabanurse_dialog_taba\n"
```

Comment:
Since you're creating a new PO msgid, you can remove the \n in the title.
**msgid "tabanurse_dialog_taba"**
if you do, then remember to update the healing center too

Reply:
Resolved